charts/puris/templates/backend-hpa.yaml

```diff
@@ -1,3 +1,24 @@
+#
+# Copyright (c) 2022-2024 Volkswagen AG
+# Copyright (c) 2022-2024 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer ISST)
+# Copyright (c) 2022-2024 Contributors to the Eclipse Foundation
```

Comment:
Minor thing but suggest to use correct year range format '2022,2024'. 
I think it applies to most of the added copyright headers in this PR.
More info: https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02

Reply:
Updated now all files with license header to this format